### ISSUE TITLE
Move governance (and similar) files to here

### ISSUE THREAD

**Issue by brosaplanella** [pybamm-team/.github#2]
At the moment we have important documents (governance, code of conduct...) in the PyBaMM repository, but they affect the whole organisation rather than a specific repository. I think it would be better to move them here, so then they can apply to all repositories within the organisation. What are your thoughts?

**Comment by valentinsulzer** [comment]
I agree, although in the long term they should move to our website

**Comment by brosaplanella** [comment]
Good point. I will move things here in the meantime. If we use Jekyll for website it will be Markdown anyway so easy to switch.
